### PR TITLE
Change of data type for storageSize from Int32 to Int64(long)

### LINKED ISSUE #61: Error while using DataManagementClient.GetFolderContentsAsync
Error message: JSON integer 2590333479 is too large or small for an Int32. Path 'included[5].attributes.storageSize', line 1, position 52380.

I've added a pull request #60 to fix this issue (Change of data type for storageSize from Int32 to Int64(long))

### PR DESCRIPTION
**Pull request by jmelhus**
Resolving error: JSON integer 2590333479 is too large or small for an Int32. Path 'included[5].attributes.storageSize', line 1, position 52380.

Fix: Change of data type for storageSize from Int32 to Int64(long)

Fixes #61

### CODE REVIEW

**Comment by sajith-subramanian** [comment]
Thanks for this PR. We will be working on updating the DM SDK shortly. Will merge this PR directly onto the DM feature branch.

**Comment by jmelhus** [comment]
Thanks @sajith-subramanian ! Approximately when will you publish a version including this fix?

**Review by VinzentHubrich — APPROVED** [review]

**Comment by VinzentHubrich** [comment]
I am experiencing the same issue. A version containing this fix would be great.

**Comment by jmelhus** [comment]
Committed another change to same repo.

Resolving bugs: Query string parameters not handled correctly as they are missing square brackets. Added for FoldersApi, HubsApi, ItemsApi, ProjectsApi and VersionsApi.

**Review by sweco-nothar — APPROVED** [review]
Please release this fix!

**Comment by tylerwarner33** [comment]
> Committed another change to same repo.
> 
> Resolving bugs: Query string parameters not handled correctly as they are missing square brackets. Added for FoldersApi, HubsApi, ItemsApi, ProjectsApi and VersionsApi.

@jmelhus I ran into this issue recently as well, thanks for submitting the PR to fix it!

File: datamanagement/source/Http/FoldersApi.gen.cs [hunk 1]
```diff
@@ -903,7 +907,7 @@ private void SetHeader(string baseName, object value, HttpRequestMessage req)
             {
                 var queryParam = new Dictionary<string, object>();
                 SetQueryParameter("filter", filter, queryParam);
```
Comment (tylerwarner33) [inline datamanagement/source/Http/FoldersApi.gen.cs:909]: @jmelhus we will also need it for the filter on the 'Search' method here.

[Documentation reference](https://aps.autodesk.com/en/docs/data/v2/developers_guide/filtering/)

**Comment by sajith-subramanian** [comment]
@jmelhus , @sweco-nothar & @tylerwarner33 - Thanks for your continued feedback and contributions! 
Apologies for the delay, since we discovered additional issues in the Data Management SDK. 
We have now released an overhauled version of the DM SDK, that includes this fix along with other enhancements. Please find it here: https://www.nuget.org/packages/Autodesk.DataManagement/2.0.0-beta3 
For sample usage please refer: https://github.com/autodesk-platform-services/aps-sdk-net/blob/main/samples/datamanagement.cshttps://github.com/autodesk-platform-services/aps-sdk-net/blob/main/samples/datamanagement.cs
Please try it out and let us know in case of any feedback! 
Thanks!

**Comment by sajith-subramanian** [comment]
Closing this PR. Please continue sharing feedback & reporting issues. Thanks!

**Pull request closed without merging**